lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+require "./lib/item.rb"
+
+class Menu
+
+  def initialize
+    @item_list = []
+  end
+
+  def item_list
+    @item_list.clone.freeze
+  end
+
+  def add_item(item)
+    @item_list.push item
+  end
+
+  def show_items
+    @item_list.each do |item|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `item`. You can omit the argument if you don't care about it. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)